openwisp_radius/migrations/0023_add_identity_verification.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.1.7 on 2021-02-26 05:29
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('openwisp_radius', '0022_organizationradiussettings_registration_enabled'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organizationradiussettings',
+            name='needs_identity_verification',
+            field=models.BooleanField(
+                blank=True,
+                default=False,
+                help_text='Whether identity verification is required at the time of'
+                ' user registration',
+                null=True,
+            ),
+        ),
+        migrations.CreateModel(
+            name='RegisteredUser',
+            fields=[
+                (
+                    'id',
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name='ID',
+                    ),
+                ),
+                (
+                    'identity_verification',
+                    models.CharField(
+                        blank=True,
+                        choices=[
+                            (None, 'No Identity Verification'),
+                            ('mobile', 'Mobile Phone'),
```

Comment:
let's put `IDENTITY_VERIFICATION_CHOICES` here instead of the raw choices, to avoid having to generate a migration every time something registers a new method

Reply:
Done (https://github.com/openwisp/openwisp-radius/pull/237/commits/c116994db22bca1cfb26a1a82fd26371bf263d96)